### DISCUSSION TITLE
Tasmota PTZ camera?

### DISCUSSION THREAD

**Discussion by dac2020** [arendst/Tasmota#17189] [General]
I'll ask this now and then in case something shows up.
I noticed this on amazon today.
https://www.amazon.com/EVERSECU-Floodlight-Waterproof-Motion-Activated-Auto-Tracking/dp/B08ZND334G

Since it's running tuya, does it mean it might be using an ESP chip?

**Comment by sfromis** [comment]
Not likely. No recent Tuya designs are using ESP chips, and a webcam is not a good candidate anyway.

**Reply by dac2020** [reply]
Ok, thanks.
Just would love to find an open source PTZ camera.

**Reply by sfromis** [reply]
If you have it, you could check the MAC....

IIRC, there exists webcams where alternate open source firmware is available (not Tasmota, unless you count the very low-end ESP32-CAM devices).

**Reply by dac2020** [reply]
Not counting the very low end cams :). Those I know about.

**Reply by sfromis** [reply]
Example of open source webcam firmware project, including recent activity: 
https://openipc.org/
https://github.com/OpenIPC/firmware

**Reply by dac2020** [reply]
Thank you. I'll take a look at those projects.
